### PR TITLE
9117 686 intro page

### PR DESCRIPTION
**Pull request by lbai001**
#### Recording
http://recordit.co/kk48Arvjzy

### CODE REVIEW

**Comment by lbai001** [comment]
please advise for any additional test needed @jbalboni

File: src/js/common/schemaform/components/AskVAQuestions.jsx [hunk 1]
```diff
@@ -7,6 +7,12 @@ function AskVAQuestions(props) {
         <div className="help-footer-box">
           <h2 className="help-heading">Need help?</h2>
           {props.children}
+          <p className="help-talk">
```
Comment (jbalboni) [inline src/js/common/schemaform/components/AskVAQuestions.jsx]: Adding this here will affect other forms. You can make a component for this content and use the `getHelp` property in the form config to pass this in and render it in line 9.

File: src/js/disability-benefits/686/containers/IntroductionPage.jsx [hunk 2]
```diff
@@ -31,11 +31,34 @@ class IntroductionPage extends React.Component {
...
+                <li>To turn in a completed Request for Approval of School Attendance (VA Form 21-674).</li>
+                <div>
+                  <a href="https://www.vba.va.gov/pubs/forms/VBA-21-674-ARE.pdf">Download VA Form 21-674</a>
+                </div>
+              </ul>
+              <div><h6>What if I need help filling out my application?</h6></div>
+              <p>An accredited representative, like a Veterans Service Officer (VSO), can help you fill out your claim.</p>
+              <div><a href="/disability-benefits/apply/help/index.html">Get help filing your claim</a></div>
+              <br/>
+              <SaveInProgressIntro
```
Comment (jbalboni) [inline src/js/disability-benefits/686/containers/IntroductionPage.jsx]: I don't think you need to use SaveInProgressIntro for this. You can use the info version of our AlertBox: https://department-of-veterans-affairs.github.io/design-system/components/detail/alertbox.html

File: src/js/common/schemaform/save-in-progress/SaveInProgressIntro.jsx [hunk 3]
```diff
@@ -15,7 +15,8 @@ class SaveInProgressIntro extends React.Component {
     let alert;
     const { renderSignInMessage, prefillEnabled, verifyRequiredPrefill, verifiedPrefillAlert, unverifiedPrefillAlert } = this.props;
     const { profile, login } = this.props.user;
-    const prefillAvailable = this.props.prefillAvailable || !!(profile && profile.prefillsAvailable.includes(this.props.formId)); // TODO: remove first clause once 526 added to list
+    const prefillAvailable = !!(profile && profile.prefillsAvailable.includes(this.props.formId));
```
Comment (jbalboni) [inline src/js/common/schemaform/save-in-progress/SaveInProgressIntro.jsx]: Where did this change come from?

Comment (lbai001) [reply src/js/common/schemaform/save-in-progress/SaveInProgressIntro.jsx]: after pulling master

File: src/js/disability-benefits/686/config/form.js [hunk 4]
```diff
@@ -18,9 +19,10 @@ const formConfig = {
   prefillEnabled: true,
   savedFormMessages: {
     notFound: 'Please start over to apply for declaration of status of dependents.',
-    noAuth: 'Please sign in again to continue your application for declaration of status of dependents.'
+    noAuth: 'Please sign in again to continue your application for declaration of status of dependents.',
```
Comment (jbalboni) [inline src/js/disability-benefits/686/config/form.js]: We can get rid of this comma

**Comment by jbalboni** [comment]
@lbai001 Can you include a screenshot of the page, too? It's easier for the content folks to review.

**Review by jbalboni — APPROVED** [review]
This looks good to me. I would like to have @peggygannon look at it too, if possible.

**Comment by lbai001** [comment]
@jbalboni 
screen shot 2018-04-23 at 10 51 47 pm
screen shot 2018-04-23 at 10 51 53 pm
screen shot 2018-04-23 at 10 52 05 pm
screen shot 2018-04-23 at 10 52 18 pm

**Review by peggygannon — CHANGES_REQUESTED** [review]
@jbalboni @lbai001 
Please see my edits below. Some of the content on this page does not reflect the latest content in the prototype. Let me know if you have any questions. Thx!
1. header: please change head to: 
Apply to add a dependent to your VA benefits
2. please update the head before the subway map to: 
Follow the steps below to add a dependent to your VA benefits.
3. Under "Prepare" please add a space btw "be" and "sure"
4. Please update the bullets under "Prepare" to match the prototype. 
5. Please update the sentence about a dependent who goes to school   "If you're claiming a dependent" to match the prototype.
6. Please update the text in blue box to match the prototype. 
7. Please move "What if I need help filing out my application..." to below blue text box to match prototype. 
8. Please delete the duplicate 4 step in the subway map.

**Review by peggygannon — APPROVED** [review]

**Comment by lbai001** [comment]
@rtluu please verify and close

**Comment by rtluu** [comment]
@lbai001 is the save in progress message at the top something that should be changed on the page itself or is that handled in a separate module?

**Pull request merged**